src/components/ScreenWrapper/index.js

```diff
@@ -72,6 +76,19 @@ class ScreenWrapper extends React.Component {
     }
 
     render() {
+        // Open the test tools menu on 5 taps in dev only
+        const isDevEnvironment = this.props.environment === CONST.ENVIRONMENT.DEV;
```

Comment:
Should this be here or still an outer layer?

I tried the taps on Login page and it didn't work? Now I am not sure if we would even need it there, but we do have controls like sign-in and language change which cannot be tested offline if the Test modal won't show up.

Reply:
That's a good point. It will be good to place it in a more outer layer.

Reply:
I added this to the ScreenWrapper since it appears to wrap almost every page. The public screens that have content other than a loading indicator, such as the SetPasswordPage and the SignInPage, do not use the screen wrapper.

I could try to use the ScreenWrapper on these pages but I would have to test how that works. Alternatively I could create another wrapper component specifically for this gesture and use it on these public pages as well in the screen wrapper.

I think it's relatively rare to test changing offline state on the public screens, so maybe I could implement that with a follow up PR sometime next week? Does that work for you guys?

Reply:
Yeah a follow up PR sounds okay to me.